docs/opening-with-new-project-system.md

```diff
@@ -1,12 +1,51 @@
 # Opening with the new project system
 
+## Background
+
+The solution (.sln) file specifies which project system should be used to open each project. For example:
+
+```
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Library3", "Library3.csproj", "{ADFEAAF5-225C-4E13-8B65-77057AAC44B8}"
+EndProject
+```
+
+Here the "Library3.csproj" project should be opened with the project system designated by the "FAE04EC0-301F-11D3-BF4B-00C04F79EFBC" GUID.
+
+While the solution file entry associates a specific project with a specific project system, ultimately this information comes from the [ProviderProjectFactoryAttribute](https://learn.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.shell.provideprojectfactoryattribute)--or the equivalent entries in a .pkgdef file--which associates a file extension (like .csproj) with a project system.
```

Comment:
Removing the culture causes a redirect to the user's requested culture where possible.

```suggestion
While the solution file entry associates a specific project with a specific project system, ultimately this information comes from the [ProviderProjectFactoryAttribute](https://learn.microsoft.com/dotnet/api/microsoft.visualstudio.shell.provideprojectfactoryattribute)--or the equivalent entries in a .pkgdef file--which associates a file extension (like .csproj) with a project system.
```